messaging-experiments.yaml

```diff
@@ -200,3 +200,118 @@
                   label: Start Browsing
                   action:
                     navigate: true
+- id: MULTI-STAGE-ABOUTWELCOME-ZERO-STAGE-TEST
+  enabled: true
+  # The filter `!('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue)` is used to exclude
+  # users who have switched channels or re-installed the browser without having a new profile
+  targeting: env.version >= '81.' && env.channel == 'default' && localeLanguageCode == "en" && ((isFirstStartup && !('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue)) || 'zero-stage-test' in activeExperiments)
+  arguments:
+    slug: zero-stage-test
+    active: true
+    features: []
+    startDate: ""
+    endDate: ""
+    proposedDuration: 0
+    proposedEnrollment: 0
+    referenceBranch: ""
+    bucketConfig:
+      count: 9000
+      start: 0
+      total: 10000
+      namespace: zero-stage-test
+      randomizationUnit: normandy_id
+    userFacingName: "Multi-Stage About:Welcome Zero Stage"
+    userFacingDescription: Test of the Zero Stage JSON flag, with and without override content
+    isEnrollmentPaused: false
+    branches:
+      - slug: msw-disabled
+        ratio: 1
+        feature:
+          featureId: "aboutwelcome"
+          enabled: false
+          value: null
+      - slug: msw-default
+        ratio: 1
+        feature:
+          featureId: "aboutwelcome"
+          enabled: true
+          value: {}
+      - slug: msw-no-titles
+        ratio: 1
+        feature:
+          featureId: "aboutwelcome"
+          enabled: true
+          value:
+            id: msw-no-titles
+            template: multistage
+            screens:
+              - id: "AW_GET_STARTED"
+                order: 0
+                content:
+                  zap: true
+                  title: Welcome to Firefox
+                  subtitle: The fast, safe, and private browser that's backed by a non-profit.
+                  primary_button:
+                    label: Start Setup
+                    action:
+                      navigate: true
+                  secondary_button:
+                    text: Have an account?
+                    label: Sign in
+                    position: top
+                    action:
+                      type: OPEN_URL
+                      addFlowParams: true
+                      data:
+                        args: https://accounts.firefox.com/?service=sync&action=email&context=fx_desktop_v3&entrypoint=activity-stream-firstrun&style=trailhead
```

Comment:
this looks old JSON as we no longer uses args

https://searchfox.org/mozilla-central/source/browser/components/newtab/content-src/lib/aboutwelcome-utils.js#88